ai/worker/container.go

```diff
@@ -87,12 +88,18 @@ func NewRunnerContainer(ctx context.Context, cfg RunnerContainerConfig, name str
 	} else {
 		hardware = hdw
 	}
+	runnerVersion := &Version{}
```

Comment:
Think would want to configure this to include the cfg.Pipeline and cfg.ModelID simialr to HardwareInformation so only the `Version` field is blank? or default version field to "0.0.0"?

Reply:
Good idea. Updated in 062135b14b1d890fb1da2080594732f488414ad6